src/node/install.sh

```diff
@@ -370,20 +378,20 @@ if [ ! -z "${ADDITIONAL_VERSIONS}" ]; then
 fi
 
 # Install pnpm
-if bash -c ". '${NVM_DIR}/nvm.sh' && type pnpm >/dev/null 2>&1"; then
-    echo "pnpm already installed."
```

Comment:
Why was this removed?

Reply:
few images like `centos:centos7` has `pnpm` pre-installed which does not support the node version 16 so we should eliminate this condition and force install pnpm with the installed npm to support the compatible version of pnpm for node 16.

Reply:
Thanks for the thoughts here, however, we shouldn't override pnpm version if it already exists. This would have side effects on other distros which are neither centos nor are attempting to install node 16 

> few images like centos:centos7 has pnpm pre-installed which does not support the node version 16 

Did you find our tests failing?

Reply:
kindly check below, if we use the default pnpm which is pre-installed in centos:7 scenaio. it is failing for pnpm -v check
```
The current version of Node.js is v16.20.2
Visit https://r.pnpm.io/comp to see the list of past pnpm versions with respective Node.js version support.


❌ pnpm check failed.
🧪 Executing duplicate test for feature 'node'...
⚠️ Skipping duplicate test for node because '/workspaces/features/test/node/duplicate.sh' does not exist.



  ================== TEST REPORT ==================
❌ Failed:      'centos-7'

@prathameshzarkar9 ➜ /workspaces/features (update-logic-for-ppa) $ docker exec -it 2f bash
[root@2f6f7f406396 /]# pnpm -v
ERROR: This version of pnpm requires at least Node.js v18.12
The current version of Node.js is v16.20.2
Visit https://r.pnpm.io/comp to see the list of past pnpm versions with respective Node.js version support.
[root@2f6f7f406396 /]# 
```

Reply:
@samruddhikhandale, reverting this change. let me know your thought on this.
adding a code to install pnpm specific version compatible for the installed node version